dali/operators/reader/loader/video/frames_decoder.cc

```diff
@@ -76,56 +75,49 @@ int64_t seek_memory_video_file(void *data_ptr, int64_t new_position, int origin)
   return memory_video_file->Seek(new_position, origin);
 }
 
-}   // namespace detail
+}  // namespace detail
 
 using AVPacketScope = std::unique_ptr<AVPacket, decltype(&av_packet_unref)>;
 
 const std::vector<AVCodecID> FramesDecoder::SupportedCodecs = {
-  AVCodecID::AV_CODEC_ID_H264,
-  AVCodecID::AV_CODEC_ID_HEVC,
-  AVCodecID::AV_CODEC_ID_MPEG4
-};
+    AVCodecID::AV_CODEC_ID_H264, AVCodecID::AV_CODEC_ID_HEVC, AVCodecID::AV_CODEC_ID_MPEG4};
```

Comment:
The previous formatting is clearer for me.

Reply:
Autoformatting... restored